components/dashboard/src/Menu.tsx

```diff
@@ -136,11 +136,9 @@ export default function Menu() {
             await Promise.all(
                 teams.map(async (team) => {
                     try {
-                        members[team.id] = usePublicApiTeamsService
-                            ? await publicApiTeamMembersToProtocol(
-                                  (await teamsService.getTeam({ teamId: team!.id })).team?.members || [],
-                              )
-                            : await getGitpodService().server.getTeamMembers(team.id);
+                        members[team.id] = publicApiTeamMembersToProtocol(
```

Comment:
As a follow-up, we can also use the types from the Public API in our models, and avoid the conversion to the types defined by server.